Version Migration Guides/V3_to_V4_ParameterMigrationGuide.md

```diff
@@ -0,0 +1,178 @@
+
+## What changed between V3 and V4
+
+The majority of the LDAR-Sim parameters from V3 still exist in V4, but many of them have been renamed or reworked. Some parameters have been removed entirely. There are many new input files users can provide to define the modelled infrastructure in more granularity and previously existing input files have mostly been removed or reworked.
+
+## Guidance on moving from V3 to V4
+
+When moving from V3 to V4, it is advised to read the updated [user manual](../USER_MANUAL.md) to get an initial understanding of the scope of all the changes that have occurred. Users will need to make new input files in order to run V4 simulations: Critically users will need a [sites file](../USER_MANUAL.md#sites-file) and the new [emissions file](../USER_MANUAL.md#emissions-file) to run any simulations. Many parameters have been changed or renamed. Reference the [Parameter File Changes](#parameter-file-changes) section for more detailed information. It may be easier for users to attempt to parameterize the V4 parameters while referencing the user manual for aid instead of attempting to "migrate" V3 simulation parameters.
+
+## Parameter File Changes
+
+### Simulation Settings
+
+#### Simulation Settings: Renamed Parameters
+
+| V3 Parameter | V4 Parameter    | Notes                                                                 |
+|--------------------|-----------|-----------------------------------------------------------------------|
+| n_processes| processes_count  | None |
+| n_simulations| simulation_count  | None |
+
+#### Simulation Settings: Removed Parameters
+
+| V3 Parameter | Notes                                                                   |
+|--------------------|-----------|
+| print_from_simulations| now always enabled  |
+| pregenerate leaks| now always enabled  |
+| outputs| functionality moved to [output parameters](../USER_MANUAL.md#6-output-settings) |
+| start_date| moved to virtual world |
+| end_date| moved to virtual world |
+
+#### Simulation Settings: New Parameters
+
+No new parameters were introduced in simulation settings.
+
+### Virtual World
+
+#### Virtual World: Renamed Parameters
+
+| V3 Parameter | V4 Parameter    | Notes                                                                 |
+|--------------------|-----------|-----------------------------------------------------------------------|
+| infrastructure_file| infrastructure: sites_file  | What was the infrastructure is now just one of the files that define infrastructure. See the [user_manual](../USER_MANUAL.md#infrastructure) on new infrastructure parameters in the virtual world for more information|
```

Comment:
Suggestion: The \`infrastructure_file\` is now just one of the files that define the virtual world. See ...